README.asciidoc

```diff
@@ -344,6 +344,15 @@ enough, `match` will be `null`.
 To use the script the previously shown build instructions need to be executed
 (including the invocation of the `symlinks` target).
 
+== History of os-autoinst
+
+At a time Bernhard M. Wiedemann from SUSE was on the openSUSE testing team and
```

Comment:
That's a lie :)

https://lizards.opensuse.org/2010/05/25/automated-opensuse-testing/ predates Bernhard's SUSE employment (compare https://www.linkedin.com/in/bernhardwiedemann/)

Reply:
ah right, he actually told me :) Will correct